Cargo.toml

```diff
@@ -199,10 +199,10 @@ number_prefix = { version = "0.4.0", default-features = false, features = ["std"
 tui = { version = "0.17.0", optional = true, default-features = false, features = ["crossterm"] }
 
 # Datadog Pipelines
-datadog-search-syntax = { path = "lib/datadog/search-syntax" }
 datadog-filter = { path = "lib/datadog/filter" }
-sha2 = { version = "0.10.2", optional = true }
+datadog-search-syntax = { path = "lib/datadog/search-syntax" }
```

Comment:
Some of these Cargo.toml changes are unrelated. I noticed we were _mostly_ alphabetising deps, and got liberal with the 'Sort Lines' feature in CLion.